mozilla/foundation.mozilla.org
# 7389 section hrs

Closes #7389 
Related PRs/issues: This PR depends on #7390. Please don't merge until that PR has been merged in.


**Link to sample test page**: https://foundation-s-7389-secti-c8kqgs.herokuapp.com/en/privacynotincluded/fine-close-tv-build-radio-end/

**Steps to test**

1. Visit the link above
2. Scroll to the "News" section, note there are now new dividers in between the requested sections.
3. If everything appears correctly, testing is complete!
